### ISSUE TITLE
Update/Re-generate OpenMetadataTypes

### ISSUE THREAD

**Issue by ilovechai** [odpi/egeria#7011]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The UML diagram https://egeria-project.org/types/5/0512-Derived-Schema-Elements/ below describes that a CalculatedValue classification can be added to a SchemaElement. 
<img width="999" alt="image" src="https://user-images.githubusercontent.com/8127448/196759408-d1de6caf-6721-46a0-8416-739d1fd07144.png">

Which should mean a RelationalTable whose superType is > SchemaAttribute > SchemaElement should be able to add the `CalculatedValue` classifiation

<img width="760" alt="image" src="https://user-images.githubusercontent.com/8127448/196759696-856016e9-9f1e-4281-9478-a3fda0c5f0f6.png">

https://egeria-project.org/types/5/0505-Schema-Attributes/?h=schema#schemaattributetype-relationship

<img width="797" alt="image" src="https://user-images.githubusercontent.com/8127448/196760406-62457f50-fa1b-45c1-9cbf-5392f6833351.png">

This is link to the supported OpenMetadataTypes https://raw.githubusercontent.com/odpi/egeria/master/content-packs/OpenMetadataTypes.json which contains the CalculatedValue ClassificationDef that supports `SchemaType` instead, needs to be updated to `SchemaElement`

```
{
"class": "ClassificationDef",
"headerVersion": 1,
"guid": "4814bec8-482d-463d-8376-160b0358e139",
"name": "CalculatedValue",
"status": "ACTIVE_TYPEDEF",
"version": 1,
"versionName": "1.0",
"category": "CLASSIFICATION_DEF",
"description": "A field within a schema that is calculated via the formula and query targets rather than stored.",
"origin": "bce3b0a0-662a-4f87-b8dc-844078a11a6e",
"createdBy": "ODPi Egeria",
"createTime": 1588261366992,
"validInstanceStatusList": [
"ACTIVE",
"DELETED"
],
"initialStatus": "ACTIVE",
"propertiesDefinition": [
{
"headerVersion": 1,
"attributeName": "formula",
"attributeType": {
"class": "PrimitiveDef",
"headerVersion": 1,
"version": 1,
"versionName": "1.0",
"category": "PRIMITIVE",
"guid": "b34a64b9-554a-42b1-8f8a-7d5c2339f9c4",
"name": "string",
"primitiveDefCategory": "OM_PRIMITIVE_TYPE_STRING"
},
"attributeStatus": "ACTIVE_ATTRIBUTE",
"attributeDescription": "Expression to create the value.",
"valuesMinCount": 0,
"valuesMaxCount": 1,
"indexable": true,
"attributeCardinality": "AT_MOST_ONE",
"unique": false
}
],
"validEntityDefs": [
{
"headerVersion": 1,
"guid": "5bd4a3e7-d22d-4a3d-a115-066ee8e0754f",
"name": "SchemaType",
"status": "ACTIVE_TYPEDEF"
}
],
"propagatable": false
}
```

### Expected Behavior

This is link to the supported OpenMetadataTypes https://raw.githubusercontent.com/odpi/egeria/master/content-packs/OpenMetadataTypes.json which contains the CalculatedValue ClassificationDef that supports `SchemaType` instead, needs to updated to be changed to `SchemaElement`

### Environment

```markdown
- Egeria: 3.10
- OS:
- Java: 11
- Browser (for UI issues):
- Additional connectors and integration:
```

**Comment by mandy-chessell** [comment]
Open Metadata Types are versioned.  

Version 1 of CalaculatedValue did connect it to SchemaType.  However this was updated to connecting to SchemaElement in Version 2.   

The way types are updated is through TypeDefPatches.    Version 1 of the type appears in the archive as a ClassificationDef.   Version 2 of this type is present in the archive as a TypeDefPatch.

When the archive is loaded by the server, the typedefs (ie the version 1 of all of the types) are loaded first.  Then the TypeDefPatches are loaded to make the changes.

I can see both versions of CalculatedValue in the archive and version 2 is correctly set up in the server when it starts.

I am not sure there is anything to change since the right values are in the archive and the latest type version is running in the server.

**Comment by mandy-chessell** [comment]
Here is the link to the structure of the archive - https://egeria-project.org/features/metadata-archiving/overview/#inside-an-open-metadata-archive

**Comment by mandy-chessell** [comment]
Yes, the repository content manager is updated with the typedef patches.   This is how the service runs normally in the OMAGServerPlatform.

I would suggest that you re-run `OpenMetadataTypesArchiveUtility` to generate `OpenMetadataTypesArchive.json` for 3.10.

This is the first time that anyone has said that they use the `OpenMetadataTypesArchive.json` which is why we update it infrequently.  The OMAGServerPlatform uses the Java call OpenMetadataTypesArchive.getOpenMetadataArchive() to get the latest types ... which may be a better solution for you

**Comment by ilovechai** [comment]
@mandy-chessell We use the OpenMetadataTypesArchive.json just for reference purpose and no where in code. Patching the igc(internal) metadata store worked for me. 
Shouldn't the internal metadata store be patched with OpenMetadataTypes automatically by the egeria repository content manager?

**Comment by mandy-chessell** [comment]
The internal type store is located within the egeria repository content manager.
